Web/jqueryAssignment1/Assignment4.js

```diff
@@ -0,0 +1,7 @@
+$(document).ready(function()
+{
+    $("tr:even").css("background-color", "#FFFFFF");
+    $("tr:odd").css("background-color", "#696969");
+    $("tr:odd").css("color", "WHITE");
```

Comment:
You should avoid this line as you are using same selector. Instead you could write like this:-
$("tr:odd").css({"color": "WHITE", "background-color":"#696969"});
Here in this u are passing more than one css properties via JSON.